Apollo.Dockerfile

```
@@ -0,0 +1,17 @@
+
+RUN npm install -g pnpm
+
+WORKDIR /app
+
+COPY package.json pnpm-lock.yaml checkNodeVersion.js ./
+COPY apollo-server ./apollo-server
+COPY tsconfig.docker.json ./
+
+RUN pnpm install
+
+RUN pnpm run compile-apollo
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)